snmp/postfixdetailed

```diff
@@ -500,10 +507,10 @@ my $data=$received."\n".
 	$rardnf."\n".
 	$rarnfqa."\n".
 	$iuscp."\n".
+	$msefl."\n".
```

Comment:
Unless we are pulling librenms/librenms#6173 completely that has already happened.